Demo0.java

```diff
@@ -1,9 +1,15 @@
 // Prints the numbers 0 to 5.
 // (Same program as shown in lecture 1-1).
-public class PrintTo5 {
+public class Demo0 {
 	public static void main(String[] args) {
 		// Declares an int variable, and sets it to 1
-        int i = 0;
```

Comment:
You could have started from i = 0, 
It would have meant you needed to increment i before printing, like this: 
while (i<6) {
             i=i+1;
            System.out.println(i);
        }